spaces/aliases.json

```
@@ -47,5 +47,6 @@
   "pillar": "pillarwallet.eth", 
   "yaxis": "yaxis.eth",
   "spaceswap": "spaceswap.eth",
-  "descience.eth": "people-dao.eth"
+  "descience.eth": "people-dao.eth",
+  "df": "dforcenetwork.eth"
```

You will need to set your actual space. Do you mean dforce space?
```suggestion
  "dforce": "dforcenetwork.eth"
```

---

So the key `dforce` is the original space without the ENS, right?

---

Yes, exactly. Your changes are good.